### DISCUSSION TITLE
how is sway started?

### DISCUSSION THREAD

**Discussion by tve** [Manjaro-Sway/manjaro-sway#350] [Q&A] [answered]
I just installed manjaro-sway, looks awesome! Thanks so much for making it available!

Q: how is sway started? I see that greetd is involved and presumably systemd-logind but I'm not finding where the commandline for sway is put together. I'd like to make it log somewhere I can inspect...

**Comment by boredland** [comment, marked as answer]
Hi there! `greetd` picks up the session files from `/usr/share/wayland-sessions`.
